libsolidity/analysis/ReferencesResolver.cpp

```diff
@@ -195,9 +196,10 @@ void ReferencesResolver::operator()(yul::FunctionDefinition const& _function)
 
 void ReferencesResolver::operator()(yul::Identifier const& _identifier)
 {
-	bool isSlot = boost::algorithm::ends_with(_identifier.name.str(), "_slot");
-	bool isOffset = boost::algorithm::ends_with(_identifier.name.str(), "_offset");
+	bool isSlot = boost::algorithm::ends_with(_identifier.name.str(), ".slot");
+	bool isOffset = boost::algorithm::ends_with(_identifier.name.str(), ".offset");
```

Comment:
By the way: Maybe at some point we could replace this by actually resolving anything with dots as a path structure and looking among ``stackSlots()`` in the last part - then we'd automatically get this as well as consistent versions for memory and calldata arrays, etc.
But that's a separate discussion, no need to do it right here, right now.
Ah yes, you even have comments for that, nice :+1:!